docs/manual/docs/cmd/spo/app/app-upgrade.md

```diff
@@ -14,7 +14,8 @@ Option|Description
 ------|-----------
 `--help`|output usage information
 `-i, --id <id>`|ID of the app to retrieve information for
-`-s, --siteUrl <siteUrl>`|Absolute URL of the site to install the app in
+`-s, --siteUrl <siteUrl>`|Absolute URL of the site to upgrade the app in
+`--scope [scope]`|Scope of the app catalog: tenant|sitecollection. Default tenant
```

Comment:
`tenant|sitecollection` should be wrapped in backticks. Currently, the pipe creates a new table column and some of the information is cut off.